core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorMaterializedViewDefinition.java

```diff
@@ -44,10 +44,10 @@ public ConnectorMaterializedViewDefinition(
             @JsonProperty("schema") Optional<String> schema,
             @JsonProperty("columns") List<Column> columns,
             @JsonProperty("comment") Optional<String> comment,
-            @JsonProperty("owner") Optional<String> owner,
+            @JsonProperty("owner") String owner,
```

Comment:
All these `@JsonProperty`s are misleading, since this is not a  `@JsonCreator`.

Reply:
added todo: https://github.com/trinodb/trino/issues/7537